### PR TITLE
Add parallax effect to photo gallery, fix margins in css

### PR DESCRIPTION
**Pull request by danyag0820**

### CODE REVIEW

File: portfolio/src/main/webapp/images/github.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="1024" height="1024" viewBox="0 0 1024 1024" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (saur-g) [inline portfolio/src/main/webapp/images/github.svg:1]: Is it possible to include a github <png> format file?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,25 @@
...
 
+    private ArrayList<String> messages = new ArrayList<String>(
+        Arrays.asList("How are you?","Doing well,you?","Funny you should ask"));
+
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Gson gson = new Gson();
+    String json = gson.toJson(messages);
+    //return json;
```
Comment (saur-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: You can remove this comment.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,25 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
```
Comment (saur-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:22]: Could you please format the file to have sorted imports.
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html [hunk 4]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```
Comment (saur-g) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: Looks like  "getRandomQuoteUsingArrowFunctions" function has been commented out. Could you please update the code.

Comment (danyag0820) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html]: done!

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -12,17 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// function getRandomQuoteUsingArrowFunctions() {
+//   fetch('/random-quote').then(response => response.text()).then((quote) => {
+//     document.getElementById('quote-container').innerText = quote;
+//   });
+// }
```
Comment (saur-g) [inline portfolio/src/main/webapp/script.js:19]: You can also remove this code, if this is not being used.

**Pull request merged**